lib_common/src/d1_common/cert/jwt.py

```diff
@@ -221,16 +244,26 @@ def validate_and_decode(jwt_bu64, cert_obj):
       cert_obj: cryptography.Certificate
         Public certificate used for signing the JWT (typically the CN cert).
 
+      leeway_sec: integer
+        Number of seconds to allow certificate to keep working before and after it has
+        expired. For use in tests and for working around inexact server clock.
+
+      disable_expiration_check: bool
+        True: Skip check for expired or not yet valid certificate.
+
     Raises:
       JwtException: If validation fails.
 
     Returns:
       dict: Values embedded in the JWT.
-
     """
     try:
         return jwt.decode(
-            jwt_bu64.strip(), cert_obj.public_key(), algorithms=["RS256"], verify=True
+            jwt_bu64.strip(),
+            cert_obj.public_key(),
+            algorithms=["RS256"],
+            verify=True,
+            leeway=HUNDRED_YEARS_SEC if disable_expiration_check else leeway_sec,
```

Comment:
Minor point: setting disable_expiration is the same as setting leeway_sec to 100 years, so why not just set the lengthy leeway in test code instead?